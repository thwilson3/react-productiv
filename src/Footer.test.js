import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

it("renders without crashing", function () {
    // this is a low-value test, but better than nothing
    render(<Footer />);
  });

  it("matches snapshot", function () {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });