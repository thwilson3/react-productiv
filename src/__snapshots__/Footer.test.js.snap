// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <footer
    class="App-footer mt-5 text-muted"
  >
    <small>
      Prødutïv is copyright ©2020 by Flüffy Data Enterprises, Inc.
    </small>
  </footer>
</div>
`;
